samples/HostConfig/cortana-beta-light.json

```diff
@@ -0,0 +1,300 @@
+{
+  "spacing": {
+    "small": 4,
+    "default": 8,
+    "medium": 12,
+    "padding": 16,
+    "large": 20,
+    "extraLarge": 32
+  },
+	"separator": {
+		"lineThickness": 1,
+		"lineColor": "#E6E6E6"
+	},
```

Comment:
Spacing consistency issue, these lines are tabs whereas the others are spaces. I think we want to use tabs in our json files.